### PR TITLE
group autofill-related prefs, re-enable storeWhenAutocompleteOff, document prefs

### PR DESCRIPTION
**Pull request by nodiscc**
Ref. https://github.com/pyllyukko/user.js/commit/5e2e5770c92d68fba9c59c0f15fd5624dca012c0

### CODE REVIEW

File: user.js [hunk 1]
```diff
@@ -500,17 +500,25 @@ user_pref("browser.cache.disk_cache_ssl",			false);
...
 // CIS Version 1.2.0 October 21st, 2011 2.5.5 Delete Download History
 // Zero (0) is an indication that no download history is retained for the current profile.
 user_pref("browser.download.manager.retention",			0);
 
-// CIS Version 1.2.0 October 21st, 2011 2.5.6 Delete Search and Form History
+// http://kb.mozillazine.org/Signon.autofillForms
+// Do not automatically fill sign-in forms with known usernames and passwords; 
+// instead allow selecting username/password from a list.
+// https://www.torproject.org/projects/torbrowser/design/#identifier-linkability
+user_pref("signon.autofillForms",
```
Comment (pyllyukko) [inline user.js]: Where's the rest of this line?

Comment (nodiscc) [inline user.js]: oops sorry.
I see you fixed it, thanks

**Comment by pyllyukko** [comment]
Thanks for the PR!

File: user.js (hunk 1 above)
Comment (pyllyukko) [inline user.js]: No worries. Did you notice how the new CI integration spotted it right away? :)

Comment (nodiscc) [inline user.js]: Yeah that was exactly my thought ("at least travis catched it...").Well done.

**Pull request merged**